rllib/examples/attention_net.py

```diff
@@ -193,26 +191,14 @@ def get_cli_args():
             obs = env.reset()
             done = False
             total_reward = 0
-            # start with all zeros as state
-            num_transformers = config["model"]["attention_num_transformer_units"]
-            attention_dim = config["model"]["attention_dim"]
-            memory = config["model"]["attention_memory_inference"]
-            init_state = state = [
-                np.zeros([memory, attention_dim], np.float32)
-                for _ in range(num_transformers)
-            ]
             # run one iteration until done
             print(f"RepeatAfterMeEnv with {config['env_config']}")
             while not done:
-                action, state_out, _ = algo.compute_single_action(obs, state)
```

Comment:
I don't think this is right. compute_single_action(obs) won't keep track of the states for you. You'd still have to manually handle state_out and passing it in as the new state. For inti_state you can use `policy.get_initial_state()` and go with that one. To see the error you can run `python rllib/examples/attention_net.py --as-test --stop-reward=70 --framework tf --no-tune` both on master and on this branch. It shows reward of greater than 70 on master, but not on this branch.

Reply:
Also this code-path is not invoked in CI as of now.

Reply:
I forgot that this is different code than with the input_from_raw_dict methods now. We should change this very soon.

Reply:
yep, but in a different PR.